src/bionics_ui.cpp

```diff
@@ -20,15 +20,117 @@ bionic_chars( "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ\"#&()*+./:;@
+{
+    static std::array< bionic_col_data, column_num_entries> col_data = { {
+            {
+                3, translate_marker( "Status" ),
+                string_format( translate_marker( "CBM Status:\n\t<color_c_green>+</color> - activated\n\t<color_c_red>x</color> - incapacitated. This CBM cannot be used for some time." ) )
+            },
+            {8, translate_marker( "Activation" ), translate_marker( "Amount of Bionic Power required to activeate this CBM." )},
+            {8, translate_marker( "Turn Cost" ), translate_marker( "Amount of Bionic Power this CBM consumes every turn while activated. Values like 1 /600 mean that 1 Power will be consumed every 600 turns." ) }
+        }
+    };
+    return col_data[col_idx];
+}
+
+void print_columnns( bionic *bio, bool is_selected, const catacurses::window &w, int right_bound,
+                     int cur_print_y )
+{
+    const bionic_data &bio_data = bio->id.obj();
+    nc_color print_col;
+
+    if( is_selected ) {
+        print_col = h_white;
+    } else if( bio->powered ) {
+        print_col = c_light_cyan;
+    } else {
+        if( bio_data.gun_bionic ) {
+            print_col = c_light_green;
+        } else {
+            print_col = c_dark_gray;
+        }
+    }
+
+
+    for( size_t col_idx = column_num_entries; col_idx-- > 0; ) {
+        std::string print_str;
+        switch( col_idx ) {
+            case    column_status: {
+                int col_w = get_col_data( col_idx ).get_width();
+                if( is_selected ) { //fill whole line with color
+                    mvwprintz( w, point( right_bound - col_w, cur_print_y ), print_col, "%*s", col_w, "" );
+                }
+                int pos_x = right_bound - 1;
+                if( bio_data.toggled && bio->powered ) {
+                    nc_color print_col2 = is_selected ? h_green : c_green;
+                    mvwprintz( w, point( pos_x, cur_print_y ), print_col2, "+" );
+                }
+                --pos_x;
+                if( bio->incapacitated_time > 0_turns ) {
+                    nc_color print_col2 = is_selected ? h_red : c_red;
+                    mvwprintz( w, point( pos_x, cur_print_y ), print_col2, "x" );
+                }
+                --pos_x;
+
+                right_bound -= col_w;
+                continue; //special printing case; skip generic handling
+            }
+            break;
+            case    column_act_cost:
+                if( bio_data.power_activate > 0 ) {
+                    print_str = string_format( "%d", bio_data.power_activate );
+                }
+                break;
+            case    column_turn_cost:
+                if( bio_data.charge_time > 0 && bio_data.power_over_time > 0 ) {
+                    print_str = bio_data.charge_time == 1
+                                ? string_format( _( "%d" ), bio_data.power_over_time )
+                                : string_format( _( "%d /%d" ), bio_data.power_over_time,
```

Comment:
Should it be marked for translation? If so, you probably need to add translator comments.